src/habitat.cr

```diff
@@ -241,7 +241,7 @@ class Habitat
 
     class HabitatSettings
       {% if type_with_habitat.superclass && type_with_habitat.superclass.type_vars.size == 0 && type_with_habitat.superclass.constant(:HABITAT_SETTINGS) %}
-        {% for decl in type_with_habitat.superclass.constant(:HABITAT_SETTINGS).map { |setting| setting[:decl] } %}
+        {% for decl in type_with_habitat.superclass.constant(:HABITAT_SETTINGS).map(&.[:decl]) %}
```

Comment:
Same. Reverting Ameba should allow you to revert this change.

Reply:
Sure! Reverted in https://github.com/luckyframework/habitat/pull/70/commits/3d3d5827cfd075c27379be6dc5813627c5cf2354